docs/API.md

```diff
@@ -13,10 +13,13 @@ The main endpoints are:
 - `/api/:integration/authentications/:authId` - To request a specific authentication
 - `/api/:integration/authentications/:authId/refresh` - To refresh a specific authentication
 
-While most actions are available through the dashboard, there are two common use cases wit Pizzly's API:
+While most actions are available through the dashboard, there are two common use cases with Pizzly's API:
 
 - saving an integration's configuration on-the-fly (`POST /api/:integration/configurations`);
 - retrieving an OAuth payload (`GET /api/:integration/authentications`).
+- delete an authentification (`GET /api/:integration/authentications/:id`).
```

Comment:
Is this meant to be a DELETE rather than a get?

Reply:
Good catch, pushed a fixed :ok_hand:  Thanks!